src/classes/RD2_EnablementDelegate_CTRL.cls

```diff
@@ -141,10 +141,12 @@ public with sharing class RD2_EnablementDelegate_CTRL extends STG_Panel {
     @AuraEnabled
     public static void enableEnhancement() {
         try {
-            STG_Panel.stgService.stgRD.isRecurringDonations2Enabled__c = true;
+            STG_Panel.stgService.stgRD.IsRecurringDonations2Enabled__c = true;
 
             if (!Test.isRunningTest()) {
                 upsert STG_Panel.stgService.stgRD;
+                UTIL_MasterSchedulableHelper.setScheduledJobs();
+                UTIL_OrgTelemetry_SVC.asyncProcessNpspSettingsTelemetry();
```

Comment:
@RadSFDeveloper I added this second line because I realized we never connected enablement to the feature parameter for telemetry. This follows the same logic we have for Customizable Rollups where the FeatureParameter is updated immediately after the setting is changed. That allows the information to be sync'd into Wilma that evening instead of waiting for the next package push to get the state of RD2.